app/controllers/profiles_controller.rb

```diff
@@ -5,10 +5,8 @@ def edit
   end
 
   def show
-    @user           = User.find_by_slug!(params[:id])
-    rubygems        = @user.rubygems_downloaded
-    @rubygems       = rubygems.slice!(0, 10)
-    @extra_rubygems = rubygems
```

Comment:
Is this removal of `@extragems_rubygems` and `rubygems` var assignments and following view change related to Rails 5 update or is it just a code cleanup/refactoring?